Firestore/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Unreleased
+- [changed] If you're manually tracking dependencies, Firestore SDK no longer
```

Comment:
This phrasing hides what actually changed. What changed should be first and how to potentially handle it should come after.

How about:

- [changed] Removed the Firestore's dependency on `Protobuf`. If you're manually tracking dependencies you may be able to remove it from your build (note, however, that other Firebase components may still require it).

Reply:
Done.